README.md

```diff
@@ -15,4 +15,4 @@ LOCAL_REPO="img2vec-pytorch" MODEL_NAME="resnet50" ./cicd/build.sh
 
 ```
 docker buildx build --platform linux/amd64,linux/arm64 -t image-2-vector:tag -f Dockerfile.distroless --push .
```

Comment:
Is this still the correct command for building here?

Reply:
Yeah, however that's just for building locally (you're going to want to put in the actual url in addition to the image name `iamge-2-vector:[tag]`